src/scaffold/ScaffoldView.tsx

```diff
@@ -78,8 +84,16 @@ const FirstPage: React.FC<PageProps> = ({ onNext }) => {
                         <img className="selectedCardTag" src={selectedItemIcon} />
                         <div className="cardTitle">
                           <p className="cardTitleTxt">{template.name}</p>
+                          {template.name !== template.packageName && (
+                            <p className="cardTitleTxt packageName">{template.packageName}</p>
+                          )}
+                        </div>
+                        <p className={`cardDescription ${template.name === template.packageName ? 'spaceTop' : ''}`}>
+                          {template.description}
+                        </p>
+                        <div className="cardFooter">
+                          <p>{template.author.name}</p>
```

Comment:
This is a bit more complicated than that.

I'd say you come up with a function to deal with it. Otherwise the logic is something like

```js
if (typeof template.author === 'string') {
  return template.author;
} else if (typeof template.author === 'object' && template.author) {
  return template.author.name;
} else {
  return '';
}
```